### DISCUSSION TITLE
Create Apps with Vala & GTK

### DISCUSSION THREAD

**Discussion by cassidyjames** [elementary/edw#7] [Ideas]
Speaker: **TBD, please comment if interested**

Learn how to create a simple app using the elementary stack: Vala, GTK, and Meson. This talk walks you through the first section of the elementary developer guide

**Comment by dxps** [comment]
This is a good idea!
@cassidyjames Are you referring to the [Writing Apps](https://docs.elementary.io/develop/writing-apps/the-basic-setup) section? I would say that - depending on the time allowed for this, of course - even say that even a sample app to illustrate that journey can help during the talk and remain as a reference for new beginners.
Btw of beginners, I am seriously planning to cover all the developer material and start contributing to this great project! So I may get back with a feedback about it here (probably by updating this comment).

**Reply by cassidyjames** [reply]
@dxps this was actually written by @danrabbit at first, and I copied it over here from a private repo. :) But I think just a straightforward talk about developing a Hello World sort of app for elementary OS would make sense, and yeah, the Developer Docs would be a good reference.
